File: Collen/python/quine.py
def fct():
    pass
def main():
    #c1
    s = "def fct():{nt}pass{n}def main():{nt}#c1{nt}s = {c}{s}{c}{nt}print(s.format(nt=chr(10)+'    ', n=chr(10), c=chr(34), s=s)){n}#c2{n}main()"
    print(s.format(nt=chr(10)+'    ', n=chr(10), c=chr(34), s=s))
#c2
main()

File: Collen/c/quine.c
#include <stdio.h>
/*
	c1
*/
int main(void){
/*
	c2
*/
  char *a="#include <stdio.h>%c/*%c	c1%c*/%cint main(void){%c/*%c     c1%c*/%cchar *a=%c%s%c;printf(a, 10, 10, 10, 10, 10,	10, 10, 34, a, 34, 10, 10);return 1;}%cint fct(void){return 1;}%c";printf(a, 10, 10, 10, 10, 10, 10, 10, 34, a, 34, 10, 10);return 1;}
int fct(void){return 1;}
